src/JuliaFormatter.jl

```diff
@@ -5,7 +5,7 @@ using Tokenize
 using DataStructures
 using Pkg.TOML: parsefile
 
-export format, format_text, format_file, DefaultStyle, YASStyle
+export format, format_text, format_file, DefaultStyle, YASStyle, DocumentFormatStyle
```

Comment:
Maybe we would just call it `VSCodeStyle`?

Reply:
I'm fine with that, although maybe `MinimalStyle` would be more informative.

Also there remains an open question of how the arguments of something like a function call should be formatted. Right now this is a derivative of YASStyle, which might be ok but some might prefer the indentation style of Default/Blue styles instead.

It should be possible to dynamically choose which style to do based on the positioning of the arguments but I haven't thought about it too much.

Reply:
Or would be indent of 4 style be preferred  (`DefaultStyle`) and then just stick with that ? assuming the argument appears on a newline since this wouldn't mess with line breaks

Reply:
Yes, `MinimalStyle` is much better, I like it!

On the indentation style for function arguments, not really sure :) Is there an easy way for me to try out the options? We could also just merge this for now and then iterate on that particular question a bit?